src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -951,15 +943,170 @@ The description of the configuration options for the remaining ProActive compone
 [[_shell_Starting_ProActive_using_Shell_scripts]]
 ==== Starting ProActive Using Shell Scripts
 
-===== ProActive server installation
 
-After configuring the appropriate settings as outlined in the previous section, execute the `proactiveInstaller.sh` script along with the `PASSPHRASE` used to encrypt the passwords.
+After configuring the necessary settings as described in the previous section, you are ready to begin the installation of the ProActive server using the `proactiveInstaller.sh` script.
```

Comment:
Did you mention that the installer script needs to be run with root privileges?

Reply:
You are right, I forgot it, I added this:

Before exploring the installation options, it's important to note that `proactiveInstaller.sh` must be executed with root privileges, and you need to grant it execution permission:

----
$ sudo chmod +x proactiveInstaller.sh
----